### DISCUSSION TITLE
Disproportionate effect of abrupt changes in terrain?

### DISCUSSION THREAD

**Discussion by IsotoCedex** [Universite-Gustave-Eiffel/NoiseModelling#592] [Q&A]
Continuing with the discussion, #585, and with the same source data, I observe a greater effect on attenuation  in NM than in other software.
It is as if a abrupt change in terrain were treated the same as a noise barrier.
What do you think?  Better behavior of NM or the commercial software?.
Here you are the scene: [map.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11930842/map.zip)

- Terrain:
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/01cc5e74-d064-4fd4-a2bc-436fab001712)

- NM Lden Result:
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/84562845-ef2d-4b6e-bf3e-96dd5e6aac54)

- Other software Lden Result:
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/e55b89aa-78c6-4767-bf60-4d25e43b991c)

- The following image shows a section of the terrain from the road to the lowest point of the terrain (first profile), the evolution of Lden with NM (second profile) and the evolution of the level with commercial software (third profile).
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/46d215dc-eacf-45dc-83c6-060319dc51ca)

**Comment by pierromond** [comment]
Maybe I am wrong, but in the case of the other commercial software, are you sure the receiver are following the ground ? (at 4 m height)

**Reply by IsotoCedex** [reply]
I think so.
I have generated the receivers manually, and applied the NM "set height" tool for the height (4m)
I have also tried with the generation of receivers by "Delaunay Grid".
The result is almost identical.

**Reply by pierromond** [reply]
For NM yes, but for commercial software too ?

**Reply by IsotoCedex** [reply]
Yes, for commercial software too. 4 m height following terrain.

**Reply by pierromond** [reply]
mmmm, we will need to find some time to look in details.. :)

**Reply by IsotoCedex** [reply]
Thank you very much. I'll keep waiting.

**Comment by IsotoCedex** [comment]
Good afternoon @pierromond and @nicolas-f 
**It seems to be a visibility issue**.
NM does not compute receivers that are not visible from the road, if the obstacle is terrain.
That is why it behaves the same, with an abrupt change in topography. Does not matter if there is an elevation gain or an elevation loss.
It does not matter if the horizontal or vertical diffraction options are activated or not, or the distance to take diffraction into account.

I have prepared a simplified model:
- Datasource: [SourceDataNM3.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11984096/SourceDataNM3.zip)
- Process: [Process.txt](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11984100/Process.txt)

I have calculated from source and from traffic, with different diffraction options.
The result with or without vertical diffraction is always the same:
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/b0c3f53f-b400-4371-b6f4-f5dc2309589b)

I have obtained the rays and also their points of origin and destination: [rays_line&vertex.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11984139/rays_line.vertex.zip)
I have checked the origin and end points of the rays. All origins are 0.05m above the ground, and all destination points are 4m above the ground.
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/1415b183-a1df-4595-b0dc-1303f7e448af)

I have calculated visibility with QGIS and, Eureka!! Receivers in visual shadow are not calculated.
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/0b083057-5a9a-46e7-8981-249fbc8e30ea)

If we raise the source to 2 meters, we calculate with NM, there are more visible points, so the propagation reaches those points.
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/8b6fdbf9-9600-4c9d-9e9a-7be5a6175838)

Rays, contours and receivers with commertcial software (source 0.05m, receivers 4 m relative height): [Results_com_Soft.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11995830/Results_com_Soft.zip)
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/4ae60fdc-b809-4657-81c1-521dd9f36b72)
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/87642224/c73167af-3b94-4e07-9192-bb29b2f4b07e)


**CONCLUSION:** I think there is a problem with vertical diffraction.
- It works on buildings, but not on terrain.
- When there is a terrain level loss, shadow receivers are not calculated.
- Terrain barriers and topographic losses behave the same way for that reason.
- Or so I think......
